tests/main.cpp

```diff
@@ -58,6 +61,11 @@ int main(int argc, char** argv)
   #endif
 
   std::cout << "Start testing " << std::endl;
+  #if defined(_WIN32)
```

Comment:
Two things it is duplicate code. Might be got introduced while rebasing. 

The next thing is that in my os `_templogs` directory was not getting deleted. The reason is it's created in `main.cpp` so the directory contains some temp files which need to go away. When I tried creating the directory in the `summary_writer_test.cpp` the issue gets resolved i.e the directory gets deleted correctly. Now the challenge is how to handle it in `summary_writer_test.cpp` file for that what we can do is to create it in the constructor of SummaryWriterTestFixture
The code would get slightly duplicated but I think we might handle it by using a static flag. Let me know what you think.

The code tried is like this
```
class SummaryWriterTestsFixture {
 
  public:
  SummaryWriterTestsFixture() {
    #if defined(_WIN32)
        _mkdir("_templogs");
    #else
        mkdir("_templogs", 0777);
    #endif
  };

 
  static mlboard::FileWriter* f1;
  static size_t currentSize;
 };

```

Reply:
Hey @jeffin143, Looks like you missed to make this change. Please make this change. The PR is ready to get merged.

Reply:
Shouldn't it be destructor Since constructor would only run when the object is created

Reply:
Oh I understood